### PR TITLE
sonic-mgmt / IXIA : T2 specific test for cisco platforms to verify ECN Counter operation 

### PR DESCRIPTION
**Pull request by sreejithsreekumaran**
### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Back port request
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [ ] 202311
- [x] 202405

#### What is the motivation for this PR?

MIGSMSFT-630 Add sonic-mgmt / IXIA test for cisco platforms to verify ECN Counter operation.

#### How did you do it?

- Create two flows one for each lossless TC per Ixia port each at half of 99.98% rate

- Inject traffic and read ECN Counters and verify it incremented.
 
- flap the interface state (shut, wait for 0.2 seconds, no shut the interfaces of Ixia)
 
- restart the traffic streams
 
- verify that the ECN counters increment from the baseline value (new values read MUST NOT be zero)

#### How did you verify/test it?

Verified on T2 DUT setup

### CODE REVIEW

**Comment by sreejithsreekumaran** [comment]
@kevinskwang this is T2 specific for #14824

**Comment by alpeshspatel** [comment]
FYI: @abdosi @sdszhang @kevinwangsk

File: tests/snappi_tests/multidut/ecn/test_multidut_ecn_marking_cisco8000.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+        testbed_config, port_config_list, snappi_ports = snappi_dut_base_config(duthosts,
+                                                                                snappi_ports,
+                                                                                snappi_api)
+
+    logger.info("Snappi Ports : {}".format(snappi_ports))
+    snappi_extra_params = SnappiTestParams()
+    snappi_extra_params.multi_dut_params.multi_dut_ports = snappi_ports
+
+    run_ecn_test_cisco8000(
```
Comment (sdszhang) [inline tests/snappi_tests/multidut/ecn/test_multidut_ecn_marking_cisco8000.py:91]: Let's change it to generic, not vendor specific.  other vendor can implement their platform check later on.

**Comment by sdszhang** [comment]
Can you make this work for t0/t1 as well.  Once multidut is merged back to t0/t1, we can delete the t0/t1 change in #14824.  

@rraghav-cisco @abdosi  for viz

**Comment by sreejithsreekumaran** [comment]
#15586 opened in lieu of this.

**Pull request closed without merging**